GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/LooseObjectStepTests.cs

```diff
@@ -73,6 +98,12 @@ private void DeleteFiles(List<string> filePaths)
             }
         }
 
+        private int CountPackFiles()
+        {
+            string[] files = Directory.GetFiles(this.PackRoot);
+            return files.Where(file => string.Equals(Path.GetExtension(file), ".pack", System.StringComparison.OrdinalIgnoreCase)).Count();

```

Comment:
split LINQ lines across multiple lines. No need to have the `files` variable, as we can chain LINQ:

```
return Directory.GetFiles(this.PackRoot)
                 .Where(file => string.Equals(Path.GetExtension(file), ".pack", System.StringComparison.OrdinalIgnoreCase))
                 .Count();
```

Reply:
and use `this.Context.FileSystem` instead of `Directory` directly?

Reply:
Cool on the linq, since we're in a functional test I don't believe we have this.Context